site/docs/configuration/expected-outputs/javascript.md

```diff
@@ -152,6 +154,14 @@ module.exports = (output, context) => {
 };
 ```
 
+This is an example of an assertion that uses data from a configuration defined in the assertion's YML file:
```

Comment:
Could you please add an example to `python.md` as well?

Reply:
Done!